pullrequest.rst

```diff
@@ -47,6 +47,10 @@ Here is a quick overview of how you can contribute to CPython:
 .. [*] If an issue is trivial (e.g. typo fixes), or if an issue already exists,
        you can skip this step.
 
+.. note::
+   In order to keep the commit history intact, please avoid force-pushing to
+   the PR.
```

Comment:
Perhaps add `Reviewers often want to look at individual commits.`  Especially for big PRs, this may apply even to the multiple commits in the initial PR.

Reply:
I added the sentence.